src/music/Drums.ts

```diff
@@ -0,0 +1,58 @@
+import { AdsrOscillatorInstrument, SampleInstrument } from "./Instruments.js";
+import { Samples, SampleNames } from "./Samples.js";
+
+export enum Drums {
+    KICK, SNARE, CLOSED_HI_HAT, OPEN_HI_HAT
+}
+
+export class Drumkit {
+
+    tri: AdsrOscillatorInstrument;
+    noise: SampleInstrument;
+
+    // TODO: generalize sampling
+    snare: SampleInstrument;
+    kick: SampleInstrument;
+
+    constructor() {
+        this.tri = new AdsrOscillatorInstrument({ type: "triangle" }, { attack: 0.01, decay: 0.2 }, 5);
+        this.noise = new SampleInstrument(Samples[SampleNames.WHITE_NOISE], { attack: 0.01, decay: 0.1 }, 1.5);
+        this.snare = new SampleInstrument(Samples[SampleNames.SNARE], { sustain: 1 }, 1.5);
+        this.kick = new SampleInstrument(Samples[SampleNames.KICK], { sustain: 1 }, 1.5);
```

Comment:
These can be default initializations, they don't have to be in the constructor

Reply:
Also, document this. What is a `tri`? Why these specific values? If they're arbitrary and they sound good, mention that.